src/query/block/types.go

```diff
@@ -104,3 +152,18 @@ type Builder interface {
 type Result struct {
 	Blocks []Block
 }
+
+// ConsolidationFunc consolidates a bunch of datapoints into a single float value
+type ConsolidationFunc func(datapoints ts.Datapoints) float64
+
+// TakeLast is a consolidation function which takes the last datapoint
+func TakeLast(values ts.Datapoints) float64 {
+	if len(values) == 0 {
+		return math.NaN()
+	}
+
+	return values[len(values)-1].Value
+}
+
+// Type assertion
+var _ ConsolidationFunc = TakeLast
```

Comment:
What is this needed for exactly?

Reply:
to ensure TakeLast actually implements ConsolidationFunc. I guess i can remove it since  multiSeriesBlock Consolidate will throw a compilation error if TakeLast doesn't implement it